### PR TITLE
Desktop Resolutions

### PR DESCRIPTION
**Pull request by shinkim0914**
Adding two libraries.

### CODE REVIEW

**Comment by lipis** [comment]
Hey @shinkim0914, we already have quite a few resolutions and maybe it would be a good idea to expand those if needed..

**Comment by dwelle** [comment]
I've checked and while we similar libs, we don't quite have desktop resolutions so I think it's fine to ship this.

I'll separate this from the lean canvas though, as there's an almost identical one, here: https://libraries.excalidraw.com/?sort=default#shellerbrand-canvases

File: libraries.json [hunk 1]
```diff
@@ -337,6 +337,19 @@
...
+    "description": "Blank desktop frames of common resolutions. Use to wireframe desktop apps.",
+    "authors": [
+      {
+        "name": "Shin Kim",
+        "url": "https://twitter.com/_shinkim"
+      }
+    ],
+    "source": "shinkim/desktop-resolutions.excalidrawlib",
+    "preview": "shinkim/desktop-resolutions.png",
+    "date": "2021-06-05"
```
Comment (dwelle) [inline libraries.json:351]:
```suggestion
    "date": "2021-06-10"
```

**Comment by dwelle** [comment]
Thanks @shinkim0914!

**Pull request merged**